__mocks__/events.js

```diff
@@ -0,0 +1,10 @@
+const events = jest.genMockFromModule('events');
```

Comment:
Can you move this under the `test` folder and switch to TypeScript with ES6 `import/export` syntax?

Reply:
According to jest's manual mocks it specifically looks for a `__mocks__` dir. But just having `jest.mock('events')` in the test script works. The resulting import is a mocked implementation. So I got rid of the `__mocks__` folder.